composer.json

```diff
@@ -16,11 +16,11 @@
         "forum": "https://discourse.zendframework.com/c/questions/components"
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^7.1"
```

Comment:
We need to confirm it, but it's quite possible that next major will support only PHP 7.2+.